client/src/components/StyleSelector/index.jsx

```diff
@@ -6,27 +6,19 @@ class StyleSelector extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = {
```

Comment:
Will you be adding state back in the future??

Reply:
If not, we can maybe change this into a functional component. Otherwise, looks good, Kanchan! 👍

Reply:
Good question Monica, I am getting rid of the state and moving it to its parent component (Overview Component)

Reply:
> If not, we can maybe change this into a functional component. Otherwise, looks good, Kanchan! 👍

Yeah that's a really good point, I am thinking about doing that too :)